setup.py

```diff
@@ -89,6 +89,7 @@ def run(self):
         "scikit-learn>=0.18",
         "scipy>=1.1.0",
         "tqdm>=4.53.0",
+        "pandas>=1.0.0",
```

Comment:
this is already added here: https://github.com/cleanlab/cleanlab/blob/8470c72667629e09b3f14850448b9f947b28b68a/setup.py#L92